packages/config/tests/load/tests.js

```diff
@@ -58,6 +58,10 @@ test('--defaultConfig can specify pinned versions', async (t) => {
   await runFixture(t, 'empty', { flags: { defaultConfig: { plugins: [{ package: 'one', pinned_version: '1' }] } } })
 })
 
+test('--defaultConfig ignores pinned versions that are empty strings', async (t) => {
+  await runFixture(t, 'empty', { flags: { defaultConfig: { plugins: [{ package: 'one', pinned_version: '' }] } } })
```

Comment:
This test tries to set `plugins[0].pinned_version` to an empty string and checks the output of `@netlify/config`.